spec/requests/doctors_spec.rb

```diff
@@ -0,0 +1,108 @@
+require 'rails_helper'
```

Comment:
Hi @OybekKayumov. I'm currently getting these when I run your tests.

![image](https://user-images.githubusercontent.com/55185309/186405326-f4cbfd26-ce07-49d5-a983-fb3a19efdd25.png)

Can you confirm to make sure they're passing as expected?